package.json

```diff
@@ -3,14 +3,14 @@
   "version": "0.0.0",
   "description": "",
   "scripts": {
-    "lint": "eslint server tests",
+    "lint": "eslint tests routes helpers",
     "flow:start": "flow start",
     "flow:stop": "flow stop",
     "pretest": "flow check",
     "test": "DEBUG=interview:* mocha --compilers js:babel-register tests/",
     "posttest": "npm run lint",
-    "start": "DEBUG=interview:* node app.js",
-    "dev": "DEBUG=interview:* nodemon app.js"
+    "start" : "DEBUG=interview:* node bin/www.js",
+    "dev": "DEBUG=interview:* nodemon bin/www.js"
```

Comment:
I dont remember, can you remind me please, why we decided to use bin/www.js to start our app? Because of conformity with express-generator project structure?